bouncer/templates/base.html.jinja2

```diff
@@ -3,18 +3,22 @@
   <head>
     <meta charset=utf-8>
 
+    {% if shared and group == '__world__'  %}
```

Comment:
Usually, this would be fine, but since we're dealing with a piece of code that could potentially leak private information in case there is a bug, maybe we should think about this a bit more.

The problem now is that you're reading the `shared` and `group` property in the view code (or code that is called by the view code) and you're returning the value to the template. The template then interprets the value and either shows the quotes or the text.

I think it would be better to only have either:

a) Only have one boolean that signals if we want to show the quote and text or not, that boolean should be assigned in the view, in which case we can test that we're checking for the correct values.

b) You were already talking about maybe going back to showing boilerplate data in cases where the annotation is private, in that case, we can make just render the `quote` and `text` variables in the template, and the view code tests can give us the confidence that we're rendering the correct piece of data depending on the input.